app/controllers/home_controller.rb

```diff
@@ -1,4 +1,5 @@
 class HomeController < ApplicationController
   def show
+    gon.availableCities = Couch.availableCities
```

Comment:
I love how this works. Do you have any idea about how JS can read this without making it an ivar?